docs/components/table.md

```diff
@@ -1028,6 +1028,116 @@ export default {
 </script>
 ```
 
+Here is another example of how to use cell slot to completely customize content inside the cell.
+
+::: tip NOTE
+You can use the `.k-table-cell-title` and `.k-table-cell-description` classes within the slot as shown in the example below to leverage preconfigured styles, should you need to make a multi-row layout. Or you could use your custom styles.
+:::
+
+<div>
+  <KTable
+    :headers="cellSlotHeaders"
+    :fetcher="cellSlotFetcher"
+    :cellAttrs="cellSlotAttrs"
+    :enableClientSort="true"
+    hidePaginationWhenOptional
+    >
+    <template v-slot:name="{row}">
+      <img class="mr-2" src="/img/kong-logomark.png" :alt="row.img.alt">
+      <div class="d-flex flex-column">
+        <span class="k-table-cell-title">{{row.name}}</span>
+        <span class="k-table-cell-description truncate">{{row.description}}</span>
+      </div>
+    </template>
+  </KTable>
+</div>
+
+```html
+<template>
+  <KTable
+    :cellAttrs="cellAttrsFn"
+    :enableClientSort="true"
+    :fetcher="fetcher"
+    :headers="headers"
+    hidePaginationWhenOptional
+  >
+    <template #name="{row}">
+      <img :alt="row.img.alt" class="mr-2" :src="row.img.src">
+      <div class="d-flex flex-column">
+        <span class="k-table-cell-title">{{ row.name }}</span>
+        <span class="k-table-cell-description">{{ row.description }}</span>
+      </div>
+    </template>
+  </KTable>
+</template>
+
+<script>
+export default {
+  data() {
+    return {
+      headers: [
+        { label: 'Name', key: 'name', sortable: true },
+        { label: 'ID', key: 'id' },
+        { label: 'Enabled', key: 'enabled' },
+        { key: 'actions', hideLabel: true },
```

Comment:
I think you need to remove this actions column since you're not actually utilizing it